bundles/org.openhab.ui/web/src/pages/settings/model/add-from-thing.vue

```diff
@@ -281,7 +282,11 @@ export default {
               groupNames: (this.parent) ? [this.parent.item.name] : []
             }
           }
-          this.ready = true
+
+          this.$oh.api.get('/rest/items').then((items) => {
```

Comment:
No need to reload the items on every thing change IMO. We can assume they will not change while the user is on that screen (not strictly true, there could be external changes in the meantime, but it's a fair assumption). Only do it if `this.items.length === 0` otherwise simply set `this.ready` to true.

There are a few reports of instances where getting the items takes seconds (#793) - admittedly a problem in itself - so displaying the spinning loader too long after picking another thing should be avoided.